parser/src/parser.cpp

```diff
@@ -170,6 +240,39 @@ int main(int argc, char* argv[])
     pHandler.getParser(parserName)->parse();
   }
 
+  //--- Add indexes to the database ---//
+
+  cc::util::createIndexes(db, binDir + SQL_DIR);
+
+  //--- Create project config file ---//
+
+  boost::property_tree::ptree pt;
+
+  if (vm.count("label"))
+  {
+    boost::property_tree::ptree labels;
+
+    for (const std::string& label : vm["label"].as<std::vector<std::string>>())
+    {
+      std::size_t pos = label.find('=');
+      labels.put(label.substr(0, pos), label.substr(pos + 1));
```

Comment:
We should warn the user if a label doesn't contain an `=` character. Moreover, I don't think we should write the label to the config file in this case.